mods/gen2/formats-data.js

```diff
@@ -40,40 +40,36 @@ exports.BattleFormatsData = {
 		},
 		tier: "NFE",
 	},
-	venusaur: {
-		randomSet1: {
-			chance: 5,
+	venusaur: { //done
+		randomSet1: { //leech seed tank
+			chance: 3,
 			item: ["leftovers"],
 			baseMove1: "leechseed", baseMove2: "synthesis",
-			fillerMoves1: ["razorleaf", "razorleaf", "hiddenpowerice"],
-			fillerMoves3: ["sleeppowder", "sleeppowder", "sleeppowder", "reflect", "lightscreen"],
+			fillerMoves1: ["razorleaf", "razorleaf", "hiddenpowerice"], //leaf for pp&stab, hpice to deter grasses from switching into seed
+			fillerMoves2: ["sleeppowder", "sleeppowder", "sleeppowder", "reflect", "lightscreen","toxic"] //misc support moves
 		},
-		randomSet2: {
-			chance: 8,
-			item: ["leftovers"],
-			fillerMoves1: ["razorleaf", "gigadrain", "gigadrain"],
-			fillerMoves2: ["hiddenpowerice", "hiddenpowerice", "hiddenpowerfire", "bodyslam"],
-			fillerMoves3: ["sleeppowder", "synthesis", "leechseed", "reflect", "lightscreen"],
-			fillerMoves4: ["sleeppowder", "synthesis", "leechseed"],
 
-		randomSet3: {
-			chance: 11,
+		//removed a tanky set here to keep things in the 4-moveset limit. it didn't seem to do anything different from the above set anyway.
+
```

Comment:
I added support for five sets anyway. It was all about avoiding unwanted move combinations